binscripts/rvm-installer

```diff
@@ -278,7 +281,8 @@ fetch_versions()
       _url=https://${_domain}/api/v3/repos/${_account}/${_repo}/tags
       ;;
   esac
-  __rvm_curl -sS ${_url} |
+  notice "Fetching ${_url}"
```

Comment:
Hmm, I am not sure if it makes sense to push that to STDERR?

Reply:
Since `rvm_install_select_and_get_version` captures all STDOUT from `$(fetch_version ...)` and interprets all of that output as a version specifier, we cannot emit warning information about GitHub failures to STDOUT; it will taint the results of the version that gets successfully retrieved from the BitBucket fallback.

Therefore neither `fetch_version` nor the `fetch_versions` function that it calls can emit warning information to STDOUT; they would have to emit to STDERR.

> If we writing to STDERR, maybe we should call this function error()?
 Can I call it `warn()`? The characteristics needed for this function are:
* It writes to STDERR.
* It does not exit.
 
Feels like `warn()` may be a good fit.  What do you guys think?

Reply:
I think STDERR is meant to include errors and not warnings? @mpapis what would u suggest to do in this case?